### PR TITLE
Integrator - replace header and footer with design system components

### PR DESCRIPTION
**Pull request by Thurfjell**
### Ticket:

- [22969](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/22969)

### Description

Added header, footer, phase banner and some layout React components from the govie design system for Integrator app.

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [ ] **Dev change**
- [ ] **Additional tests**
- [ ] **Documentation**
- [ ] **Other**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This change might impact the developer experience of others and should be communicated

### Screenshots:

N/A

### CODE REVIEW

File: apps/integrator/messages/ga.json [hunk 1]
```diff
@@ -1,3 +1,9 @@
 {
- 
+  "Header": {
+    "english": "English",
+    "irish": "Gaelige",
+    "logout": "Logout",
+    "title": "Integrator",
```
Comment (alfonsograziano) [inline apps/integrator/messages/ga.json:6]: Same here. Also, at the moment we're not actively creating `ga` translations, so we can revert this

Comment (Thurfjell) [reply apps/integrator/messages/ga.json:6]: Np

File: apps/integrator/app/[locale]/(hosted)/admin/journeys/configure/[journeyId]/[stepId]/page.tsx [hunk 2]
```diff
@@ -3,8 +3,8 @@ import { notFound, redirect } from "next/navigation";
 import { AuthenticationFactory } from "../../../../../../../../libraries/authentication-factory";
 import { PageWrapper } from "../../../../../PageWrapper";
 import { JourneyEditor } from "../../../../../../../../libraries/journeyEditor";
-import Link from "next/link";
 import { errorHandler } from "../../../../../../../utils/errorHandler";
+import Link from "next/link";
```
Comment (alfonsograziano) [inline apps/integrator/app/[locale]/(hosted)/admin/journeys/configure/[journeyId]/[stepId]/page.tsx:7]: Leftover

File: apps/integrator/app/[locale]/(hosted)/admin/journeys/configure/[journeyId]/done/page.tsx [hunk 3]
```diff
@@ -2,11 +2,12 @@ import { getTranslations } from "next-intl/server";
 import { notFound } from "next/navigation";
 import { AuthenticationFactory } from "../../../../../../../../libraries/authentication-factory";
 import { PageWrapper } from "../../../../../PageWrapper";
-import Link from "next/link";
 import { errorHandler } from "../../../../../../../utils/errorHandler";
 import InputField from "../../../../../../../components/InputField";
 import CopyLink from "../../../../../../../components/CopyBtn";
 import ds from "design-system";
+import React from "react";
```
Comment (alfonsograziano) [inline apps/integrator/app/[locale]/(hosted)/admin/journeys/configure/[journeyId]/done/page.tsx:9]: Leftovers

File: apps/integrator/app/[locale]/(hosted)/admin/journeys/submissions/details/[submissionId]/page.tsx [hunk 4]
```diff
@@ -230,13 +231,13 @@ export default async ({ params: { locale, submissionId } }: Props) => {
                     {t("paymentDetails.submissionLink")}
                   </dt>
                   <dt className="govie-summary-list__value">
-                    <a
+                    <Link
```
Comment (alfonsograziano) [inline apps/integrator/app/[locale]/(hosted)/admin/journeys/submissions/details/[submissionId]/page.tsx:234]: Same here

File: apps/integrator/app/[locale]/(hosted)/admin/journeys/configure/[journeyId]/[stepId]/page.tsx (hunk 2 above)
Comment (Thurfjell) [reply apps/integrator/app/[locale]/(hosted)/admin/journeys/configure/[journeyId]/[stepId]/page.tsx:7]: From what?????

Comment (Thurfjell) [reply apps/integrator/app/[locale]/(hosted)/admin/journeys/configure/[journeyId]/[stepId]/page.tsx:7]: Does the import order matter?

**Review by alfonsograziano — APPROVED** [review]
LGTM

**Pull request merged**